somecomfort/__main__.py

```diff
@@ -107,7 +107,7 @@ def _main(session):
     string_things = ['fan_mode', 'system_mode']
     bool_things = ['cancel_hold', 'permanent_hold']
     settable_things = {float: number_things, str: string_things}
-    readonly_things = ['current_temperature', 'equipment_output_status']
+    readonly_things = ['current_temperature', 'current_humidity', 'equipment_output_status', 'outdoor_temperature', 'outdoor_humidity']
```

Comment:
Ideally this would be properly wrapped and indented. Bonus points if you submit another PR to fix it :)